makeup-example/Makefile

```diff
@@ -8,93 +8,65 @@
 # required during development.
 #
 # The commands are:
-#   make up 	- bring up all containers
-#   make stop 	- stop all containers
-#   make reset 	- stop all containers, update them and any dbs, and restart
-#   make test	- run any local tests
-#   make logs   - follow docker logs from the containers
-#   make status - show status of the environment
+#   make up 
+#   make stop
+#   make reset
+#   make test
+#   make logs
+#   make status
```

Comment:
Should we include `make help` here then? Or simply refer people to `make help` for a full list of targets? It *is* self-documenting after all :-)

Reply:
done @achton

Reply:
I thought @achton meant only documenting `make help` and refer to that for getting the full list of targets and descriptions.